videos/nappn_2021/walkthrough.Rmd

```diff
@@ -0,0 +1,340 @@
+    - R packages: raster, dplyr, ggplot2
+4. Download Dryad data
+    - Download manually at [https://datadryad.org/stash/dataset/doi:10.5061/dryad.4b8gtht99](https://datadryad.org/stash/dataset/doi:10.5061/dryad.4b8gtht99) by clicking blue "Download dataset" button OR
+    - Download with R using code below
+
+```{r, eval=FALSE}
+local_path <- "dryad_data.zip"
+download.file(url = "https://datadryad.org/stash/downloads/file_stream/624637", destfile = local_path)
+unzip(local_path)
+```
+
+## Data overview
+
+The purpose of this walkthrough is to provide an introduction the TERRA REF public domain high throughput phenomics datasets (LeBauer et al 2020), and to give learners a chance to access and explore a representative cross section of these data. 
+
+In addition to this workshop, we have online tutorials for using these data at [https://terraref.github.io/tutorials/](https://terraref.github.io/tutorials/). There are also materials from previous walkthrough workshops at [https://github.com/terraref/tutorials/tree/master/videos](https://github.com/terraref/tutorials/tree/master/videos). We are striving to make these data as accessible and usable as possible. 
+
+TERRA REF is a project funded by the Advanced Research Projects Agency for Energy (ARPA-E). The objective is to advance the science and technology of high throughput field phenotyping, which is the automated measurement of plants. TERRA REF has deployed an exceptionally large robot capable of collecting data from a range of sensors at 1 mm spatial resolution every other day. You can learn more about the project from the website, terraref.org. The suite of sensors include visible light cameras, hyperspectral cameras, a laser 3d scanner, and environmental data. In addition there are weather stations and high resolution genome sequences for hundreds of varieties of Sorghum being monitored. 
+
+### Experimental design
+
+(Show picture of site) Field site is in Arizona. Field is passed over by large robot called a gantry. Lot of equipment in hanging box to collect many types of data. 
+
+Gantry goes systematically over entire field once a day. Some sensors take data every day, like 9,000 images from RGB camera. Others more intermittently, like hyperspectral images, because of data space limits. 
+
+Sensors include: 
+
+* camera that takes pairs of red-green-blue images (**Stereo RGB**)
+* thermal infrared images (**FLIR**)
+* images at a bunch of wavelengths to get hyperspectral data (**VNIR/SWIR**)
+* laser that collects points on plant surfaces to create 3D image (**3D Laser**)
+* measures plant fluorescence (**PS II Fluor**)
+* handful of others, including environmental data such as temperature and light
+
+Collecting data since 2015, and are up to 8 seasons worth in that time. Originally for sorghum, but now open to other crop species and organizations that want to use system.
+
```

Comment:
Provide core details of season 4 and 6 experiments ... that these are currently in the public domain,  are Sorghum & etc, that they are trials of the biomass assoc. panel, planting and harvest dates, # of plots. Can lean on README and documentation for this information.